src/advanced/bitcoin-anchoring.md

```diff
@@ -308,40 +269,38 @@ reasons:
 - periodic key rotation
 - changing the validators list: add/replace/remove some validators
 
-Both procedures require disabling the old anchoring keys and adding the new ones.
+Both procedures require disabling the old anchoring keys and adding the new
+ones.
 Additionally, as the anchoring Bitcoin address is a derivative from the list of
 anchoring public keys, it should be changed accordingly. The pub-keys list
 is stored in the global configuration; it can be updated by out-of-band
-means, for example, using [Configuration Update
-service](configuration-updater.md). The following properties should be taken
- into account:
+means, for example, using
+[Configuration Update service](configuration-updater.md). The following
+properties should be taken into account:
 
 1. New configuration is spread over nodes. It is still not active.
 2. New configuration has an additional parameter that indicates the height when
-  this configuration should be applied. The height is selected by the
-  administrator. It is recommended to choose the height so that the
-  configuration is applied in ~3-6 hours after it is sent into the Exonum
-  blockchain.
+   this configuration should be applied. The height is selected by the
+   administrator.
 3. After the indicated height is reached, the new configuration is applied by
-  every validator simultaneously. The list of validators is finally changed.
+   every validator simultaneously. The list of validators is finally changed.
 
 !!! warning
-    It is important that the pause between configuration appearing and
-     applying is big enough. It should be defined in accordance
-    with the necessary number of confirmations for the latest LECT.
+    It is important that the interval between the moment a new configuration
+    appears and the height when it will be applied, should be sufficient for
+    validators to vote for the new configuration.
```

Comment:
Also user should check that the network has enough funds to create transition transaction.